### DISCUSSION TITLE
Tiles OneWayCollicion

### DISCUSSION THREAD

**Discussion by lurgx** [godotengine/godot-proposals#4331] [2D]
I do not speak much English I hope you understand me The use of tilemaps is great they weigh less but a function that I would like to see in the tilemaps would be **OneWayCollicion** this would make the process faster for those of us who use tiles a lot, since we have to use several nodes for the oneway and this saturates the memory of opgl2 computers with 2gbram like my laptop and many Latinos who use the engine cannot afford the luxury of spending ram
It would be great if this could be implemented and easier ways to interact with tiles

Thank you very much for the engine it is great and easy to use and the documentation is more understandable than in other engines :)

**Comment by Calinou** [comment]
One-way collision is already implemented in the 2D physics system, and it can already be used from TileMaps: https://godotengine.org/qa/69624/how-to-trigger-oneway-collision-on-a-tilemap

**Reply by lurgx** [reply]
It is good to know, I thought it was not available in these versions, thanks for the information, I will pass on the knowledge if someone asks.
